app/controllers/ExclusionListController.scala

```diff
@@ -164,40 +164,65 @@ class ExclusionListController @Inject()(
               } yield {
                 selectedValue match {
                   case ControllersReferenceDataCodes.FORM_TYPE_NINO =>
-                    Ok(
-                      ninoExclusionSearchFormView(
-                        taxYearRange,
-                        isCurrentTaxYear,
-                        iabdTypeValue,
-                        formMappings.exclusionSearchFormWithNino,
-                        empRef = request.empRef))
+                    Redirect(
+                      routes.ExclusionListController.showExclusionSearchForm(isCurrentTaxYear, iabdType, "nino")
+                    )
                   case ControllersReferenceDataCodes.FORM_TYPE_NONINO =>
-                    Ok(
-                      noNinoExclusionSearchFormView(
-                        taxYearRange,
-                        isCurrentTaxYear,
-                        iabdTypeValue,
-                        formMappings.exclusionSearchFormWithoutNino,
-                        empRef = request.empRef))
-                  case "" =>
                     Redirect(
-                      routes.ExclusionListController.withOrWithoutNinoOnPageLoad(isCurrentTaxYear, iabdTypeValue))
+                      routes.ExclusionListController.showExclusionSearchForm(isCurrentTaxYear, iabdType, "no-nino"))
+                  case "" =>
```

Comment:
`case _` would be better incase theres a situation where it's not empty for whatever reason

Reply:
Agreed, will update this